en/getting-started/basics/editor-panels/properties.md

```diff
@@ -12,49 +12,70 @@ What displays at the right side of the node activating switch is node name, whic
 
-The attribute column that needs to be assigned value by dragging nodes will show white label. There might display `Node` on the label, which means any node can be dragged into it. Also, the label may display component names like `Sprite`,`Animation`, etc., then users need to drag the nodes to which corresponding components are mounted.
+The property column that needs to be assigned value by dragging nodes will show white label. There might display `Node` on the label, which means any node can be dragged into it. Also, the label may display component names like `Sprite`,`Animation`, etc., then users need to drag the nodes to which corresponding components are mounted.
 
-The attribute column that needs to be assigned value by dragging assets will show blue labels. There might display the asset type on the label, such as `sprite-frame`, `prefab`, `font`, etc.. The assignment can be finished as long as you drag the assets of corresponding type from **Assets**.
+The property column that needs to be assigned value by dragging assets will show blue labels. There might display the asset type on the label, such as `sprite-frame`, `prefab`, `font`, etc.. The assignment can be finished as long as you drag the assets of corresponding type from **Assets**.
 
 ![reference types](inspector/reference_type.png)
 
-  **Note**: that script file is also a type of asset. Therefore the script asset reference attribute used by components shown on the top of the above picture is also represented by a blue label.
\ No newline at end of file
+  **Note**: That script file is also a type of asset. Therefore the script asset reference property used by components shown on the top of the above picture is also represented by a blue label.
+
+## Editing several assets at once
+
+Selecting an asset in the **Assets** panel displays the properties of the asset in the **Properties** panel for query and editing. Asset settings related, see the [Asset workflow](../../../asset-workflow/index.md#common-asset-workflow) for details.
+
+When you need to edit several assets properties of the **same type** at once, you can press Shift in the **Assets** panel to select several assets, and then the **Properties** panel will display the number of selected assets and editable asset properties. After that, click the **Apply** button on the top right.
+
+![](inspector/edit_many_res.png)
+
+If the properties in the **Properties** panel show the following statuses, it means that the property value of the property in the selected several assets is inconsistent. You can choose whether to continue to modify the property in batches as required.
+- Check box is **gray**
+- Input box display **-**
+- Select box is **blank**
+
+**Note**:
+- Currently, the Material and the Texture Compression of Texture do not support.
```

Comment:
- Currently, this feature does not support Material properties and Texture Compression properties.